services/movie.py

```diff
@@ -6,9 +7,14 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: str = None
 ) -> QuerySet:
     queryset = Movie.objects.all()
 
+    if title:
+        queryset = queryset.filter(title__icontains=title)
+        return queryset
```

Comment:
The early return statement here will cause the function to return immediately after filtering by title, ignoring any genre or actor filters. Consider removing this return statement to allow further filtering by genres and actors.